components/cam/src/physics/cam/physpkg.F90

```diff
@@ -1922,6 +1927,8 @@ subroutine tphysbc (ztodt,               &
     !
 
     type(physics_ptend)   :: ptend            ! indivdual parameterization tendencies
+    type(physics_ptend)   :: ptend_tmp1
+    type(physics_ptend)   :: ptend_tmp2
```

Comment:
Both ptend_tmp* are not used here (perhaps they were used during debugging....but not used anymore)

Reply:
You are correct, my bad, will fix.